umm-spec-lib/src/cmr/umm_spec/json_schema.clj

```diff
@@ -221,23 +225,29 @@
 (defn- validate-umm-json-search-result
   "Validates the UMM JSON search result and returns a list of errors if invalid."
   [json-str concept-type schema-name umm-version]
-   (let [schema-url (umm-schema-resource concept-type schema-name umm-version)]
-     (if schema-url
-       (let [java-schema-obj (js-validations/parse-json-schema-from-uri schema-url)]
-         (js-validations/validate-json java-schema-obj json-str))
-       [(format "Unable to load schema [%s] with version [%s]." schema-name umm-version)])))
+  (let [schema-url (umm-schema-resource concept-type schema-name umm-version)]
+    (if schema-url
```

Comment:
I know it's not your code. could you change it to if-let?

Reply:
Sure. Done.